packages/backend/src/modules/dashboard/dashboard.repository.ts

```diff
@@ -1,15 +1,49 @@
-import { CreateTaskCommandDTO } from 'shared'
-import DashboardModel from './Dashboard'
+import { CreateTaskCommandDTO, CreateDashboardCommand } from 'shared'
+import mongoose from 'mongoose'
+import { Dashboard as DashboardModel } from './dashboard.model'
 import { Task } from '../task/task'
+import { Dashboard } from './Dashboard'
 
 export class DashboardRepository {
-  private readonly _dashboardModel = DashboardModel
+  private readonly _dashboardModel
 
-  constructor(boardModel: any) {
-    this._dashboardModel = boardModel
+  constructor(dashboardModel: any) {
+    this._dashboardModel = dashboardModel
   }
 
-  async addNewTaskToDashboard(createTaskCommand: CreateTaskCommandDTO, savedTask: Task) {
+  async createDashboard(createDashboardCommand: CreateDashboardCommand): Promise<Dashboard> {
+    console.log('Creating new dashboard in DB')
+    const dashboard = await this._dashboardModel.create({
+      _id: new mongoose.Types.ObjectId(),
+      title: createDashboardCommand.title,
+      description: createDashboardCommand.description,
+      imageCoverUrl: createDashboardCommand.coverImageUrl,
+      createdAt: new Date(),
+      status: createDashboardCommand.status,
+      users: [],
+      columns: []
+    })
+
+    await dashboard.save()
+    return dashboard
+  }
+
+  async getDashboards(): Promise<Dashboard[]> {
+    const dashboards: Dashboard[] = await this._dashboardModel.find()
+    console.log(dashboards)
+    return dashboards
+  }
+
+  async getDashboard(_id: string): Promise<Dashboard> {
+    const dashboard: Dashboard = await this._dashboardModel.findById(_id)
+    console.log(dashboard)
+    return dashboard
+  }
+
+  async addNewTaskToDashboard(
+    createTaskCommand: CreateTaskCommandDTO,
+    savedTask: Task
+  ): Promise<Task> {
     const dashboard = await this._dashboardModel.findById(createTaskCommand.idDashboard)
     const idCol = { idColumn: createTaskCommand.idColumn }
```

Comment:
I know, it's not yours, but if we don't use it as an object, than there is no need to create one. And there is a rule - leave the code in better state than it was 😄